assets/stylesheets/common/header.scss

```diff
@@ -0,0 +1,10 @@
+.bb-header{ 
```

Comment:
Are these styles needed to overwrite some discourse styles for h1 tags only, @kyuukestu @Alteras1? Or is this an artifact of the previous implementation?

Reply:
I believe this is an artifact of the previous implementation, I used the styles as they currently exist on RPNation including fonts, size, etc. 

I believe these styles could be added directly inside header.js via the style attribute, however after looking at some other examples the use of an external stylesheet seemed to be standard so I wanted to have my code be closer to what already exists.

Reply:
Let's remove this. Since we've decided to match the built in discourse header style, we should avoid this.

Reply:
Understood!